zh/deploy-on-aws-eks.md

```diff
@@ -360,7 +349,7 @@ spec:
 
 最后使用 `kubectl -n tidb-cluster apply -f tidb-cluster.yaml` 更新 TiDB 集群配置。
 
-更多可参考 [API 文档](https://github.com/pingcap/tidb-operator/blob/master/docs/api-references/docs.md)和[集群配置文档](configure-a-tidb-cluster.md)完成 CR 文件配置。
+更多可参考 [API 文档](https://github.com/pingcap/tidb-operator/blob/v1.1.6/docs/api-references/docs.md)和[集群配置文档](configure-a-tidb-cluster.md)完成 CR 文件配置。
```

Comment:
This doc is the `dev` version, not `release-1.1`. Is this alright to point users to `v1.1.6` API references?

Reply:
For issue #828, we should modify the doc in the `release-1.1` branch.

Reply:
I will fix #828 on `release-1.1` directly.